habitat-baselines/habitat_baselines/config/rearrange/tp_srl_oracle_nav.yaml

```diff
@@ -5,22 +5,36 @@ defaults:
   - /habitat_baselines: habitat_baselines_rl_config_base
```

Comment:
Is this config file needed? Seems like we can achieve the same through this call:
```
python -u habitat_baselines/run.py \
  --exp-config habitat_baselines/config/rearrange/rl_hl_srl_onav.yaml \
  --run-type eval \
  habitat_baselines/rl/policy=hierarchical_tp_noop_onav
```

As described in https://github.com/facebookresearch/habitat-lab/tree/train_hl/habitat-baselines/habitat_baselines

Reply:
True, I refactored the configs and removed this.